chart/infra-server/static/flavors.yaml

```
@@ -1024,6 +1024,106 @@
     - name: cluster-console-password
       description: The password to login at the openshift console
 
+#########################
+#  Openshift IBM ROKS   #
+#########################
+- id: ibmroks
+  name: IBM ROKS
+  description: IBM ROKS cluster on VPC infrastructure.
+  availability: stable
+  workflow: configuration/workflow-openshift-ibmroks.yaml
+  parameters:
+    - name: name
+      description: cluster name
+      value: example1
+      help: |
+        The name must start with a letter, can contain letters, 
+        numbers, periods (.), and hyphen (-), and must be 35 
+        characters or fewer. Use a name that is unique across 
+        regions. The cluster name and the region in which the 
+        cluster is deployed form the fully qualified domain name 
+        for the Ingress subdomain. To ensure that the Ingress 
+        subdomain is unique within a region, the cluster name 
+        might be truncated and appended with a random value within 
+        the Ingress domain name.
+        <https://cloud.ibm.com/docs/openshift?topic=openshift-cluster-create-vpc-gen2&interface=cli>
+
+    - name: nodes
+      description: number of nodes (ACS may not start if memory < 48gb total)
+      value: "3"
+      kind: optional
+
+    - name: instance-type
+      description: node machine type (ACS may not start if cores < 8 each or memory < 48gb total)
+      value: cx2.8x16
+      kind: optional
+      help: |
+        <https://cloud.ibm.com/docs/openshift?topic=openshift-vpc-flavors>
+        Current ACS(<=4.3) fails to start on ROKS instances with fewer
+        than 8 cores individually or memory less than 48GB combined.
+        ACS 4.2.2 successfully started on instance types below (minimum node count)
+        + cx2.8x16 (3)
+        + bx2.8x32 (2)
+        + mx2.8x64 (2)
+        + cx2.16x32 (2)
+        + bx2.16x64 (2)
+
+    - name: zone
+      description: IBM Cloud zone to deploy into (Example eu-de-1, us-south-1, jp-tok-1).
```

```suggestion
      description: IBM Cloud zone to deploy into
```

`help` text already contains examples, also the text in the description is camelCased, so `eu-De-1`, `us-South-1`, which probably won't work.